packages/forms/resources/views/components/date-time-picker.blade.php

```diff
@@ -88,6 +88,10 @@
                 x-ref="panel"
                 x-cloak
                 x-float.placement.bottom-start.offset.flip.shift="{ offset: 8 }"
+                @if (! $getStateBindingModifiers())
+                    wire:ignore.self
+                    wire:key="reactiveDateTimeInput-{{ $this->id }}"
```

Comment:
What happens if there are multiple date time inputs on the same component? Because then the wire:key will be the same. Maybe use `$getStatePath()`?

Reply:
Is $this->id not the id of the component?

Reply:
It's the ID of the Livewire component, no?

Reply:
`$this` in all views is always the Livewire component

Reply:
Hmm, but `$this->id` is used as `wire:key` in more places.

Reply:
Yup, but in both cases there's only one chance of it being included on the Livewire component at a time, you can only have 1 table at a time. So it's fine in those places